packages/semaphore-group-pcd/test/SemaphoreGroupPCD.spec.ts

```diff
@@ -67,7 +67,7 @@ describe("semaphore group identity should work", function () {
 
     // change merkle root to make it invalid
     const pcd = await prove(args);
-    pcd.proof.proof.merkleTreeRoot = "0";
+    pcd.proof[0] = "1";
 
```

Comment:
As this code will be likely be referenced in building other PCD, I think we should set stronger norms around testing. For example, one norm we could set is to take a valid PCD and test changing every field in claim to make sure the old proof does not verify with the new claim.

Reply:
OK, i've made this test work by updating the claim rather than the proof